pages/exhibitions/_exhibition/index.vue

```diff
@@ -80,6 +80,13 @@
       mainContent() {
         if (this.text === undefined) return;
         return marked(this.text);
+      },
+      optimisedImageUrl() {
+        return this.$options.filters.optimisedImageUrl(
+          this.heroImage.url,
+          'image/jpeg',
+          { width: 800, height: 800 }
+        );
```

Comment:
As far as I can see, this computed property "optimisedImageUrl " is not used in this file. 
Which makes me wonder if the actual issue has been fixed. The image in the og:image is still the full hero image.

Reply:
this was being applied previously in the head, not sure how it got reverted, possibly after pulling master when the GraphQL changes were made. I've fixed the both files and tested that if the social image is set on a page it is used for the social sharing image.